setup.cfg

```diff
@@ -57,6 +57,18 @@ no_implicit_optional = True
 # TODO disallow_subclassing_any = True
 disallow_untyped_decorators = True
 warn_unused_configs = True
+show_error_codes = True
+show_column_numbers = True
+exclude = (?x)(
```

Comment:
![image](https://user-images.githubusercontent.com/14194980/176497844-516964ca-4abb-4333-b1c0-de2fc14dcee9.png)
@justinchuby I am seeing this in the CIs and my local. Is it not supported by older mypy? (ONNX uses 0.782)

Reply:
Hmm, let me test that with 0.782